ChangeLog.d/psa_ecc_macros.txt

```diff
@@ -0,0 +1,4 @@
+API changes
+   * Rename PSA_ECC_CURVE_xxx to PSA_ECC_FAMILY_xxx in line with PSA spec
```

Comment:
Please put a final stop at the end of the sentence.

Reply:
Done.